opm/autodiff/FlowMainEbos.hpp

```diff
@@ -700,6 +777,9 @@ namespace Opm
         Grid& grid()
         { return ebosSimulator_->gridManager().grid(); }
 
+        const Grid& globalGrid()
+        { return *globalGrid_; }
```

Comment:
IMO returning something akin `grid().sequentialGrid()` would be more elegant. I'm not sure how much effort it is to implement this in `CpGrid`, though.